vtex-apps/store-components
# Feature/availability subscriber

#### What problem is this solving?

This solution aims to solve the impossibility of changing the name and email placeholder text values in the availability subscriber block

#### How to test it?

Click on the link below to see the implementation. The values of the name and email placeholder are set by the block properties "namePlaceholder" and "emailPlaceholder". The block documentation was updated with the descriptions of the properties added.

https://avtjoao--avantivtexio.myvtex.com/variacao-tamanho/p?skuId=205

#### Screenshots or example usage:

![image](https://user-images.githubusercontent.com/88465655/128397198-6a2ceb3c-17a3-4414-97cc-6c4296115abc.png)

![image](https://user-images.githubusercontent.com/88465655/128397296-096a33ac-90b4-4a02-9250-1ef618210648.png)


#### Describe alternatives you've considered, if any.



#### Related to / Depends on



#### How does this PR make you feel? [:link:](http://giphy.com/)



![](put .gif link here - can be found under "advanced" on giphy)
